test/e2e/packetcapture_test.go

```diff
@@ -586,6 +606,24 @@ func runPacketCaptureTest(t *testing.T, data *TestData, tc pcTestCase) {
 	if !packetCaptureStatusEqual(pc.Status, tc.expectedStatus) {
 		t.Errorf("CR status not match, actual: %+v, expected: %+v", pc.Status, tc.expectedStatus)
 	}
+
+	if tc.expectedStatus.NumberCaptured == 0 {
+		return
+	}
+	// verify packets.
+	antreaPodName, err := data.getAntreaPodOnNode(nodeName(0))
+	require.NoError(t, err)
+	fileName := fmt.Sprintf("%s.pcapng", tc.pc.Name)
+	dstFileName := filepath.Join(os.TempDir(), fileName)
+	packetFile := filepath.Join("/tmp", "antrea", "packetcapture", "packets", fileName)
+	err = data.copyPodFiles(antreaPodName, "antrea-agent", "kube-system", packetFile, os.TempDir())
+	require.NoError(t, err)
+	file, err := os.Open(dstFileName)
+	require.NoError(t, err)
+	defer file.Close()
+	defer os.Remove(dstFileName)
```

Comment:
this will remove the file before closing it?
is that an issue?

Reply:
yeah, you are right. updated.